### PR TITLE
High level app summary

### PR DESCRIPTION
**Pull request by proteusvacuum**
High level app summary, like this: https://confluence.dimagi.com/display/UATBC/Modules+and+Forms

@benrudolph 
(FYI @czue )

### CODE REVIEW

File: corehq/apps/app_manager/views/__init__.py [hunk 1]
```diff
@@ -21,7 +21,11 @@
     download_xform,
     DownloadCCZ,
 )
-from corehq.apps.app_manager.views.app_summary import AppSummaryView, DownloadCaseSummaryView
+from corehq.apps.app_manager.views.app_summary import (
```
Comment (dimagimon) [inline corehq/apps/app_manager/views/__init__.py]: F401 'AppSummaryView' imported but unused
F401 'DownloadAppSummaryView' imported but unused
F401 'DownloadCaseSummaryView' imported but unused

**Comment by czue** [comment]
took you long enough. :trollface:

**Comment by czue** [comment]
is this only available as an export? might be cool to also add to app summary (not blocking)

**Review by benrudolph — APPROVED** [review]

**Comment by benrudolph** [comment]
👠

**Comment by proteusvacuum** [comment]
@czue, I agree - I think this would make a nice page (added to my backlog)

**Comment by amsagoff** [comment]
@proteusvacuum can I get a little more info for announce? Like, how do you access this :) thx

**Comment by czue** [comment]
@amsagoff I'll just send out a quick response to your email

**Pull request merged**